wrappers/rust/icicle-core/src/ntt/mod.rs

```diff
@@ -422,20 +422,41 @@ macro_rules! impl_ntt_bench {
       $field:ident
     ) => {
         use icicle_core::ntt::ntt;
+        use icicle_core::ntt::get_root_of_unity;
+        use icicle_core::ntt::initialize_domain;
         use icicle_core::ntt::NTTDomain;
+
         use icicle_cuda_runtime::memory::HostOrDeviceSlice;
+        use icicle_cuda_runtime::device_context::DeviceContext;
         use std::sync::OnceLock;
+        use std::iter::once;
 
         use criterion::{black_box, criterion_group, criterion_main, Criterion};
         use icicle_core::{
             ntt::{FieldImpl, NTTConfig, NTTDir, NttAlgorithm, Ordering},
-            traits::ArkConvertible,
         };
 
         use icicle_core::ntt::NTT;
         use icicle_cuda_runtime::memory::HostSlice;
         use icicle_core::traits::GenerateRandom;
         use icicle_core::vec_ops::VecOps;
+        use std::env;
+
+        fn get_min_max_log_size(min_log2_default: u32, max_log2_default: u32) -> (u32, u32) {
+
+            fn get_env_log2(key: &str, default: u32) -> u32 {
+                env::var(key).unwrap_or_else(|_| default.to_string()).parse().unwrap_or(default)
+            }
+
+            let min_log2 = get_env_log2("MIN_LOG2", min_log2_default);
+            let max_log2 = get_env_log2("MAX_LOG2", max_log2_default);
+
+            assert!(min_log2 >= min_log2_default);
```

Comment:
Why limiting here?

Reply:
It that case let's skip these benches or edit the error message to be more clear

Reply:
added error messages